outer.c

```diff
@@ -291,7 +292,8 @@ void outer_helper_spawn(struct outer_helper *helper)
 	/* Notify sibling that we're done persisting their proc files
 	   and/or changing their [ug]id map */
 	int ok = 1;
-	write(pipefds_in[1], &ok, sizeof (ok));
+	size_t count = write(pipefds_in[1], &ok, sizeof (ok));
```

Comment:
nit: use ssize_t and explicitly cast it in the assert -- the CI job (and myself, occasionally) enable conversion warnings to detect mishandled overflows, and this would add a false positive.